examples/clusterscoped/escape-to-host-si-sib-mixed.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: intent.security.nimbus.com/v1
+kind: SecurityIntent
+metadata:
+  name: escape-to-host
+spec:
+  intent:
+    id: escapeToHost
+    description: "A attacker can breach container boundaries and can gain access to the host machine"
+    action: Block
+    params:
+      exclude_resources: ["app:nginx"]
+---
+
+# resources which are having matchLables defined, will not incorporate the exclusion labels
```

Comment:
We need to support exclusion labels if customer has a strategy of a cluster-wide baseline profile, and exclusion for certain names/namespaces/labels